frontend/src/components/pages/accountPage.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```

Comment:
Shouldn't these file names be AccountPage, InformationPage, and LandingPage to match capitalization of the routing?